project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 
 ////////////////////////////////////////////////////////////////////////////////
 
-addSbtPlugin("io.7mind.izumi.sbt" % "sbt-izumi" % "0.0.60")
+addSbtPlugin("io.7mind.izumi.sbt" % "sbt-izumi" % "0.0.61")
 
```

Comment:
I guess this is redundant

Reply:
@VladPodilnyk Fixed in https://github.com/7mind/izumi/pull/1189

Reply:
I've rebased the branch.